Dockerfile

```diff
@@ -5,13 +5,18 @@ LABEL description="Simple and full-featured mail server using Docker"
 ARG DEBIAN_FRONTEND=noninteractive
 ENV PYTHONUNBUFFERED=1
 
+RUN apt-get update && apt-get install -y -q --no-install-recommends curl \
```

Comment:
I think you can move these install & update steps into this Docker image: https://github.com/mailserver2/debian-mail-overlay/blob/master/Dockerfile

What do you think @AndrewSav , @SaraSmiseth ?

Reply:
We do install & update steps here too..

https://github.com/mailserver2/mailserver/blob/45649a8ac2d7eb9542d8f8bb55f4f132a875fbe7/Dockerfile#L13

Reply:
@sknight80 if we want this PR, I think this repo is the right place as opposed to `debian-mail-overlay`, because we already are getting dovecot here, albeit from a different package source. It is not clear to me what benefits are of moving it there. I'd leave it here so that we do minimal changes.  `debian-mail-overlay` is mostly for stuff that are compiled from sources. Please let me know if it makes sense.

Reply:
Yes, make sense.